homework/Stupns/homework9.py

```diff
@@ -0,0 +1,29 @@
+import timeit
+
+
+def numberGenerator(n):
+    number = 0
+    while number < n:
+        yield number
+        number += 1
+
+
+def simple_func():
+    a = []
+    b = a.append(0)
```

Comment:
Do you like this code? Two mistakes:
1) You do not need create new variable to append something into list. `.append()` method always returns None
2) Why you didn't use cycle?